### ISSUE TITLE
Move "Project Settings" into tab "General" (also default "Export Settings")

### ISSUE THREAD

**Issue by mz-memoryshuffle** [olivierkes/manuskript#1188]
There is the usual "Edit" / "Settings..." for program settings (global setup, like GUI skin, save/load behavior, Logging behavior etc.)

But the project-specific settings could be moved into the Tab "General", under the basic information.

Also, the project settings could include the "Compile Options" GUI (the left side of the compile dialog). These default compile settings would be saved as default on project level. Also, the user should be able to save multiple compile profiles.
The compile dialog would allow to select one of the default profiles, or go back to the compile option pane.

**Comment by mz-memoryshuffle** [comment]
When these settings would be moved to the project level, this would of course require a new version of fileformat (version==2).

This move could be used, to also remember the state of the application on a project level.
Right now, when switching between projects, the GUI is not re-initialized, so this leads to two problems:
* some controls still have old text in it (eg. the "note/references" contained old text from the other project, until i opened a new scene).
* when I have a complex outline, i like that my last working state is saved. so some folders may be open in the tree, some others closed. But right now, the folder tree is collapsed, after switiching to another project.
all this could be remembered on project level, so wen re-opening one can start work just were one left off.

**Comment by TheShadowOfHassen** [comment]
I was planning to change the general tab to an overview tab and add things like "tip of the day" and tracking your writing statistics. I don't think our Ideas are compatible.

However, the settings Idea is good, but I'd like to keep it in the settings menu.

I also really like the idea for compile settings or even an auto compile feature so that whenever I quit the project I can send an already compiled doc or pdf to my alpha readers.

**Comment by mz-memoryshuffle** [comment]
Cool, I like the idea of this new Overview tab!
Sounds good to have all important information about the project status here.
BTW, the book's language could be added here as information (and then preset the spell check language).

Also, why do you think project settings should be in the separate settings dialog? I would find it easier, if the Edit/Settings-Dialog is purely about the application-specific stuff, technical configuration on a global level. While project specific settings (including the compile settings) are very much part of each project itself, and would IMHO deserve their own Tab.

**Comment by TheShadowOfHassen** [comment]
I think there already is language on overview.

It could have a tab, but personally I've thought of the tabs as all the nity gritty writing stuff and the dialogs have the tools/ settings.

**Comment by mz-memoryshuffle** [comment]
I don't have "Language" in "Overview"... (v0.15.0)

Yes, makes sense too, have Tabs purely about the work itself. 
Maybe, my idea was more about separating
a) Technical App Configuration, like GUI skin, GUI language, save/load behavior (global level)
b) Project Settings, like Label, State, Revisions, Editor layout and maybe new: Compile Settings (on per project basis)

**Comment by TheShadowOfHassen** [comment]
Yes, separating them would work. I can do that in the GTK rewrite.
